src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.FSharp.xaml

```diff
@@ -33,6 +33,26 @@
                 HelpUrl="https://docs.microsoft.com/dotnet/fsharp/language-reference/compiler-options"
                 Category="General" />
 
+  <EnumProperty Name="FSharpPreferNetFrameworkTools"
+              DisplayName="Framework tools"
+              Description="Compiler to use when building the project"
+              Category="General">
```

Comment:
You're missing `HelpUrl`s for your new properties. If you can make them fwlinks, that is preferred.

Reply:
We don't have any docs for these yet, unfortunately.

Reply:
Could you make an issue for it so it isn't forgotten? Also, adding a comment above the properties would let us know that it wasn't *intentionally* forgotten. 😉

Reply:
Done, will add it to xaml file too
https://github.com/dotnet/fsharp/issues/12102